prisma/seed.js

```diff
@@ -235,6 +235,172 @@ async function seed() {
       }
     ]
   })
+
+  const iterator = 2
+
+  const curriculums = []
+  const createCurriculum = async () => {
+    const newCurriculum = await prisma.curriculum.create({
+      data: {
+        name: 'Javascript',
+        description:
+          'Learn the JavaScript fundamentals you will need for front-end or back-end development'
+      }
+    })
+    curriculums.push(newCurriculum)
```

Comment:
Don't push the created curriculum to an array, instead return it from the function, and have something else add it to an array. 

Adding to an array from inside a function is a side-effect, and it is always best to try and reduce them as much as possible

Reply:
Updated